samples/bot-conversation-sso-quickstart/js/.gitignore

```diff
@@ -21,3 +21,23 @@
 npm-debug.log*
```

Comment:
Better to commit the original `env` folder, refer to https://github.com/formulahendry/teamsfx-migrator/tree/main/templates/bot-sso/env

Reply:
added